src/py/aspen/cli/db.py

```diff
@@ -37,31 +37,28 @@ def interact(ctx, profile):
 
 
 @db.command("import-covidhub-project")
+@click.option("--covidhub-aws-profile", type=str, required=True)
```

Comment:
wouldn't this always be the biohub aws profile?

Reply:
or maybe i'm missing what you mean by `covidhub-aws-profile` exactly, or what the issue is that this PR is fixing?

Reply:
It is always the biohub aws profile.  We can't hardcode it though because everyone could name it differently.

Reply:
makes sense